### ISSUE TITLE
AlbumArt size?

### ISSUE THREAD

**Issue by jompa68** [jishi/node-sonos-http-api#609]
Is it possible to decide how big AlbumArt picture i want to see? I would like to have the possibillity to choose 128x128 size

something like /getaa?widht=128&height=128&s=1&u=x-sonos-spotify%3aspotify%253atrack%253a4Yzoi1w1a7eDOfiRN7ZDZx%3fsid%3d9%26flags%3d8224%26sn%3d1

**Comment by jishi** [comment]
Nope, that is just a proxy into the sonos player and how they serve
image's. Would need to implement rescaling which is a lot of work. Usually
you can control the rendering size when showing it?

**Comment by jompa68** [comment]
By default it shows 640x640 and i cannot find any other way to rendering size

**Comment by jishi** [comment]
I was talking about controlling the size of rendering where you display it. Building an image resampler/resizer feels like outside the scope of this project :)
